model_name is dkvmn, dataset_name is nips_task34
Read data from processed file: ../data/nips_task34/test_sequences.csv_-1.pkl
file path: ../data/nips_task34/test_sequences.csv, qlen: 1855, clen: 1855, rlen: 1855
Read data from processed file: ../data/nips_task34/test_window_sequences.csv_-1.pkl
file path: ../data/nips_task34/test_window_sequences.csv, qlen: 116155, clen: 116155, rlen: 116155
Read data from processed file: ../data/nips_task34/test_question_sequences.csv_-1_qtest.pkl
file path: ../data/nips_task34/test_question_sequences.csv, qlen: 8435, clen: 8435, rlen: 8435
Read data from processed file: ../data/nips_task34/test_question_window_sequences.csv_-1_qtest.pkl
file path: ../data/nips_task34/test_question_window_sequences.csv, qlen: 119994, clen: 119994, rlen: 119994
has test_question_file!
Start predicting model: dkvmn, embtype: qid, save_dir: saved_model/nips_task34_dkvmn_qid_saved_model_42_0_0.2_200_0.001_50_1_1_f69ea553-a44a-41d8-ad58-849fbed8d53c, dataset_name: nips_task34
model_config: {'dropout': 0.2, 'dim_s': 200, 'size_m': 50}
data_config: {'dpath': '../data/nips_task34', 'num_q': 948, 'num_c': 57, 'input_type': ['questions', 'concepts'], 'max_concepts': 2, 'min_seq_len': 3, 'maxlen': 200, 'emb_path': '', 'train_valid_original_file': 'train_valid.csv', 'train_valid_file': 'train_valid_sequences.csv', 'folds': [0, 1, 2, 3, 4], 'test_original_file': 'test.csv', 'test_file': 'test_sequences.csv', 'test_window_file': 'test_window_sequences.csv', 'test_question_file': 'test_question_sequences.csv', 'test_question_window_file': 'test_question_window_sequences.csv', 'train_valid_original_file_quelevel': 'train_valid_quelevel.csv', 'train_valid_file_quelevel': 'train_valid_sequences_quelevel.csv', 'test_file_quelevel': 'test_sequences_quelevel.csv', 'test_window_file_quelevel': 'test_window_sequences_quelevel.csv', 'test_original_file_quelevel': 'test_quelevel.csv', 'dataset_name': 'nips_task34'}
Traceback (most recent call last):
  File "/home/zhjy/lizz/Exp_DataSavePath/pykt/pykt-toolkit-main/examples/wandb_predict.py", line 140, in <module>
    main(params)
  File "/home/zhjy/lizz/Exp_DataSavePath/pykt/pykt-toolkit-main/examples/wandb_predict.py", line 80, in main
    testauc, testacc = evaluate(model, test_loader, model_name, 'None', save_test_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhjy/lizz/anaconda3/envs/pykt/lib/python3.11/site-packages/pykt/models/evaluate_model.py", line 108, in evaluate
    y = model(cc.long(), cr.long())
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhjy/lizz/anaconda3/envs/pykt/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhjy/lizz/anaconda3/envs/pykt/lib/python3.11/site-packages/pykt/models/dkvmn.py", line 67, in forward
    (w.unsqueeze(-1) * Mv[:, :-1]).sum(-2),
     ~~~~~~~~~~~~~~~~^~~~~~~~~~~~
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 1.91 GiB (GPU 0; 23.65 GiB total capacity; 4.06 GiB already allocated; 1.78 GiB free; 5.98 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF